src/pages/StatusPage.vue

```diff
@@ -34,6 +34,14 @@
                     </div>
                 </div>
 
+                <div class="my-3">
+                    <label for="auto-refresh-interval" class="form-label">{{ $t("Refresh Interval") }}</label>
+                    <input
```

Comment:
I don't know if a help text is necessary or if a refresh interval is self-explanatory.

What are your thoughts on adding something like this?

> The status page will refresh every {0} seconds

Reply:
I think it is pretty much self explanatory. But i can make this change for sure

Reply:
Description added